vue-app/src/views/Profile.vue

```diff
@@ -130,10 +132,14 @@ export default class NavBar extends Vue {
   projects: Project[] = []
   balanceBackgroundColor = '#2a374b'
   isLoading = true
+  ens: string | null = null
 
   async created() {
     this.isLoading = true
     await this.loadProjects()
+    // if (this.currentUser) {
+    //   this.ens = await ensLookup(this.currentUser.walletAddress)
```

Comment:
This probably shouldn't be commented out 😄

Reply:
you're right, it should be gone 😆... this is stored in the state now